src/main/webapp/WEB-INF/pages/messages.ftl

```diff
@@ -0,0 +1,137 @@
+                                </div>
+                                <#if messages??>
+                                    <ul class="event-list">                                         
+                                        <@createMail />
+                                    </ul>
+                                </#if>     
+                            </div>
+                        </div>                        
+                        <div id="compose" class="tab-pane fade">
+                            <div class="panel-body">
+                                <p><@spring.message 'MESSAGE.COMPOSE.TEXT'/></p>
+                                <div class="contact-form col-md-6 col-xs-12">
+                                    <form>
+                                        <div class="form-group">
+                                            <div class="form-control message-form" disabled>To: Market Time</div>
+                                            <@createInputSection 'subject' />
+                                            <textarea class="form-control message-form" id="comment" name="comment" placeholder="comment" rows="10"></textarea>
+                                            <div class="textarea-tooltip tooltip-wrapper">
+                                                <span id="comment-tooltip" class="error-tooltip"></span>
+                                            </div>
+                                            <button type="submit" id="message-btn" class="btn btn-default"><@spring.message 'MESSAGE.COMPOSE.SEND.BUTTON'/></button> 
+                                        </div>
+                                    </form>
+                                </div> 
+                            </div>
+                        </div>
+                    </div> 
+                </div> 
+            </div>
+        </div>
+    </div>
+</section>
+
+<#macro createInputSection fieldId>
+    <#local fieldName = buildName(fieldId) >
+    <#local fieldKey = buildKey(fieldId) >
+    <#assign hasError=false/>
+    <#if validationErrors?? && validationErrors["${fieldName}"]??>
+        <#assign hasError=true/>
+    </#if>  
+        <div>
+            <input <#if !hasError && fieldValue?? && fieldValue != "">value="${fieldValue}"</#if> class="form-control message-form <#if hasError>has-error</#if>" name="${fieldName}" id="${fieldId}" type="text" placeholder="${fieldName}">
+            <div class="tooltip-wrapper">
+                <span id="${fieldId}-tooltip" class="error-tooltip"><#if hasError><@spring.message '${validationErrors["${fieldName}"]}'/></#if></span>
+            </div>
+        </div>   
+</#macro>
+
+<#macro createMail>
+    <#list messages as receivedMessage>
+        <li>
+            <div class="details-container">                                         
+                <div class="panel message-details">
+                    <div class="panel-heading email-hover">
```

Comment:
nu vad rostul unei clase numite email-hover, care are apoi specificat in css un comportament pentru actiunea de hover in sine